### PR TITLE
koordlet: report aggregate node usage in NodeMetric

### PR DESCRIPTION
**Pull request by zwzhang0107**
Signed-off-by: 佑祎 <zzw261520@alibaba-inc.com>

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/statesinformer/config.go [hunk 1]
```diff
@@ -53,4 +56,6 @@ func (c *Config) InitFlags(fs *flag.FlagSet) {
 	fs.UintVar(&c.KubeletReadOnlyPort, "kubelet-read-only-port", c.KubeletReadOnlyPort, "The read-only port for the kubelet to serve on with no authentication/authorization. Default: 10255.")
 	fs.DurationVar(&c.NodeTopologySyncInterval, "node-topology-sync-interval", c.NodeTopologySyncInterval, "The interval which Koordlet will report the node topology info, include cpu and gpu")
 	fs.BoolVar(&c.DisableQueryKubeletConfig, "disable-query-kubelet-config", c.DisableQueryKubeletConfig, "Disables querying the kubelet configuration from kubelet. Flag must be set to true if kubelet-insecure-tls=true is configured")
+	fs.DurationVar(&c.MetricReportInterval, "report-interval", c.MetricReportInterval, "Deprecated since v1.1, use ColocationStrategy.MetricReportIntervalSeconds in config map of slo-controller")
+	fs.BoolVar(&c.EnableNodeMetricReport, "enable-node-metric-report", c.EnableNodeMetricReport, "Enable status update of node metric crd.")
```
Comment (eahydra) [inline pkg/koordlet/statesinformer/config.go:60]: Should the newly introduced parameters be synchronized to config/manager/koordlet.yaml?

**Comment by FillZpp** [comment]
fixes #856

File: pkg/koordlet/statesinformer/config.go (hunk 1 above)
Comment (zwzhang0107) [reply pkg/koordlet/statesinformer/config.go:60]: not necessary, just use the default value is enough

**Pull request merged**